pkg/logql/metrics.go

```diff
@@ -116,6 +116,7 @@ func RecordRangeAndInstantQueryMetrics(
 		"query_type", queryType,
 		"range_type", rt,
 		"length", p.End().Sub(p.Start()),
+		"start_delta", time.Since(p.Start()).Round(time.Minute),
```

Comment:
should we log `end_delta` if the intention is to easily find whether the query touches the last x hours of data or not?
Or maybe there is a way to add `length` to `start_delta` to find that out?

Reply:
@sandeepsukhani done